newsubprojects.md

```diff
@@ -0,0 +1,183 @@
+
+The possible recommendations of the Steering Council wil be:
+
+* Integration into an existing official Suproject.
+* Incorporation as a new official Subproject.
+* Further internal or external incubation (see below) with recommendations on steps the team can
+  take to prepare for future incorporation.
+* Rejection. This option will be used if the Steering Council thinks the proposed Subproject is not
+  progressing or will never be appropriate for graduation. If a rejected Subproject has been
+  incubating under the [jupyter-incubator](https://github.com/jupyter-incubator) GitHub
+  organization, its repository will be removed from that organization after a transition period.
+
+
+### Incorporation
+
+When an existing, external Subproject is incorporated as a new official Subproject, the following
+steps will be taken:
+
+1. The repository will be transfered over to the main Jupyter GitHub organization.
+2. A GitHub team will be created for the Subproject, with the Subproject team having
+   read/write permissions on the Subproject repository.
+3. The team will send an Email the main Jupyter list with an announcement about the new
+   Subproject.
+4. The standard Jupyter LICENSE file will be added to the repository.
+
+
+## Incubation of subprojects
+
+Incubation refers to the process of a Subproject initially being developed outside the official
+Jupypter organization. Incubation is recommended for new Subprojects with the following
+characteristics:
+
+* Significant unanswered technical questions or uncertainties that require exploration.
+* Entirely new directions, scopes or ideas that haven't been vetted with the community.
+* Significant, already existing code bases where it is not clear how the project will
+  integrate with the rest of Jupyter.
+
+Incubation also allows the broader community can easily distinguish between stable, officially
+supported and maintained Subprojects in the main GitHub organization and new efforts that are
+emerging and being developed for future inclusion.
+
+If there are questions about whether incubation is appropriate for a given Subproject, interested
+parties should simply start a discussion on the main Jupyter list to get community and Steering
+Council feedback.
+
+
+### Incubation in the [jupyter-incubator](https://github.com/jupyter-incubator) organization
+
+If a Subproject team knows from the start that it would eventually like its Subproject to be part
+of the official Jupyter organization, it can be incubated in the
+[jupyter-incubator](https://github.com/jupyter-incubator) organization.
+
+The goals of the incubation in the [jupyter-incubator](https://github.com/jupyter-incubator) organization are as follows:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while
+  complying with individual and organizational contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and
```

Comment:
"gather feedback **early and often** that will help" and delete the next bullet point